src/components/scenes/SettingsScene.tsx

```diff
@@ -155,6 +165,33 @@ export class SettingsSceneComponent extends React.Component<Props, State> {
     this.props.isLocked ? this.handleUnlock() : navigation.navigate('passwordRecovery', {})
   }
 
+  handleContactsAccess = (): void => {
+    if (this.state.contactsPermission) {
+      // App will restart after denying the permission
+      openSettings().catch(e => console.error(e))
```

Comment:
As mentioned in the engineering meetings, `console` error handling is banned. Only `showError` is acceptable, unless you have a documented reason to do it otherwise.